pyiron_base/job/script.py

```diff
@@ -31,6 +31,99 @@ class ScriptJob(GenericJob):
         project (ProjectHDFio): ProjectHDFio instance which points to the HDF5 file the job is stored in
         job_name (str): name of the job, which has to be unique within the project
 
+    Simple example:
+        Step 1. Create the notebook to be submitted, for ex. 'example.ipynb', and save it -- Can contain any code like:
+                ```
+                import json
+                with open('script_output.json','w') as f:
+                    json.dump({'x': [0,1]}, f)  # dump some data into a JSON file
+                ```
+
+        Step 2. Create the submitter notebook, for ex. 'submit_example_job.ipynb', which submits 'example.ipynb' to the
+                pyiron job management system, which can have the following code:
+                ```
+                from pyiron_base import Project
+                pr = Project('scriptjob_example')  # save the ScriptJob in the 'scriptjob_example' project
+                scriptjob = pr.create.job.ScriptJob('scriptjob')  # create a ScriptJob named 'scriptjob'
+                scriptjob.script_path = 'example.ipynb'  # specify the PATH to the notebook you want to submit.
+                ```
+
+        Step 3. Check the job table to get details about 'scriptjob' by using:
+                ```
+                pr.job_table()
+                ```
+
+        Step 4. If the status of 'scriptjob' is 'finished', load the data from the JSON file into the
+                'submit_example_job.ipynb' notebook by using:
+                ```
+                import json
+                with open(scriptjob.working_directory + '/script_output.json') as f:
+                    data = json.load(f)  # load the data from the JSON file
+                ```
+
+    More sophisticated example:
+        The script in ScriptJob can also be more complex, e.g. running its own pyiron calculations.
+        Here we show how it is leveraged to run a multi-core atomistic calculation.
+
+        Step 1. 'example.ipynb' can contain pyiron_atomistics code like:
+                ```
+                from pyiron_atomistics import Project
+                pr = Project('example')
+                job = pr.create.job.Lammps('job')  # we name the job 'job'
+                job.structure = pr.create.structure.ase_bulk('Fe')  # specify structure
+
+                # Optional: get an input value from 'submit_example_job.ipynb', the notebook which submits
+                #   'example.ipynb'
+                external_input = pr.get_external_input()
```

Comment:
can just use `pr.data.number_of_cores` as of #192

Reply:
Done.